scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -254,6 +260,45 @@ function doImportEclipse() {
   "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.ant.core.antRunner -buildfile "${DEVON_IDE_HOME}/scripts/lib_script/import.xml" -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${ECLIPSE_JAVA_HOME}/bin/java" -vmargs -DdevonImportPath="${importPath}" -DdevonImportWorkingSet="${importWorkingSets}" ${ECLIPSE_VMARGS} 
 }
 
+# $1: id
+# $2: optional plugin URL (update site)
+function doMirrorUpdateSite() {
+  if [ -z "${1}" ]
+  then
+    doFail "Parameter \"id\" is missing."
+  fi
+  local url
+  if [ -n "${2}" ]
+  then
+    url="${2}"
+  else
+    local file="${SETTINGS_PATH}/eclipse/plugins/${1}.properties"
+    if [ -f  "${file}" ]
+    then
+      plugin_id=""
+      plugin_url=""
+      doLoadProperties "${file}"
+      if [ -z "${plugin_id}" ] || [ -z "${plugin_url}" ]
+      then
+        doFail "Invalid eclipse plugin config: ${file}"
+      fi
```

Comment:
Can we avoid copy&paste and code duplication by moving this to a function?

Reply:
done